File: JS/Resto2.test.js
/**
 * @version JAVASCRIPT
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ node NomeDaClasse.test
 */
const TEST = require('tape')
const OBJ = require('./Resto2.js')

TEST('Starting Resto2 test...', (t) => {
    //Test 1
    OBJ.printAllRest2(20)

    t.end()
})
 

File: GO/Resto2_test.go
/**
 * @version GO
 * @author MARIO SAM <eu@mariosam.com.br>
 * @see I would love to work with you instead solving web code tests: hire me!
 * $ go test
 */
package GO

import "testing"

func TestResto2(t *testing.T) {
	tables := []struct {
		X int
	}{
		{2},
		{20},
	}

	for _, table := range tables {
		printAllRest2(table.X)
	}
}
